### PR TITLE
[release] update changelog

### PR DESCRIPTION
**Pull request by liuwenran**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+- Support mscoco dataset.(#1520)
+- Improve Chinese documents.(#1532)
+- Type hints.(#1481)
+- Update download link.(#1554)
+- Update deployment guide.(#1551)
+
+**Bug Fixes**
+
+- Fix documentation link checker.(#1522)
+- Fix ssim first channel bug by.(#1515)
```
Comment (plyfager) [inline docs/en/changelog.md:28]:
```suggestion
- Fix ssim first channel bug.(#1515)
```

File: README.md [hunk 2]
```diff
@@ -106,14 +106,13 @@ hope MMEditing could provide better experience.
...
-- Unifies interfaces of all components based on [MMEngine](https://github.com/open-mmlab/mmengine).
-- Support patch-based and slider-based image and video comparison viewer.
-- Support image colorization.
+A brand new version of [**MMEditing v1.0.0rc5**](https://github.com/open-mmlab/mmediting/releases/tag/v1.0.0rc5) was released in 03/01/2023:
+
+- Support well-known text-to-image method [Stable Diffusion](configs/stable_diffusion/README.md)!
+- Support an efficient image restoration algorithm [Restormer](configs/restormer/README.md)!
+- Support a new text-to-image algorithm [GLIDE](projects/glide/configs/README.md)!
+- Support swin based image restoration algorithm [SwinIR](configs/swinir/README.md)!
+- [Projects](projects/README.md) is opened for community to add projects to MMEditing.
```
Comment (plyfager) [inline README.md:115]: I suggest keepping some previous 1.x features.

File: docs/en/changelog.md (hunk 1 above)
Comment (liuwenran) [reply docs/en/changelog.md]: resolved.

File: README.md (hunk 2 above)
Comment (liuwenran) [reply README.md:115]: resolved.

**Review by plyfager — APPROVED** [review]

**Pull request merged**